sklearn/dummy.py

```diff
@@ -67,7 +67,7 @@ class DummyClassifier(MultiOutputMixin, ClassifierMixin, BaseEstimator):
     classes_ : ndarray of shape (n_classes,) or list thereof
         Class labels for each output.
 
-    n_classes_ : int or list of int
+    n_classes_ : int or list thereof
```

Comment:
no need to use thereof in such simple cases.

Reply:
Thanks! I have changed it back.